Sources/Tools/MaxConvert/hsMaterialConverter.cpp

```diff
@@ -4751,7 +4761,7 @@ static int ICompareDoneLayers(const plLayerInterface* one, const plLayerInterfac
     return 0;
 }
 
-static int ICompareDoneMats(const hsMaterialConverter::DoneMaterialData* one,
+static bool ICompareDoneMats(const hsMaterialConverter::DoneMaterialData* one,
                             const hsMaterialConverter::DoneMaterialData* two)
```

Comment:
Either `hsMaterialConverter::IEquivalent` needs to be updated to support this new format (it currently compares the result to `0` for "equal"), or the `std::sort` needs to be updated to properly handle a 3-way comparison.